### PR TITLE
`ROIPooling`, `ROIAlign` layer tests to API2.0

### PR DESCRIPTION
**Pull request by vurusovs**

### CODE REVIEW

File: src/tests/functional/plugin/shared/include/single_op_tests/roi_align.hpp [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+#include "shared_test_classes/single_op/roi_align.hpp"
+
+namespace ov {
+namespace test {
+TEST_P(ROIAlignLayerTest, Inference) {
+    run();
+}
+
+TEST_P(ROIAlignV9LayerTest, Inference) {
```
Comment (vurusovs) [inline src/tests/functional/plugin/shared/include/single_op_tests/roi_align.hpp:15]: @iefode please, note that the test isn't instantiated. Who should enable it?

File: src/tests/functional/shared_test_classes/src/single_op/roi_align.cpp [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+}
+
+static int randInt(int low, int high) {
+    std::random_device rd;
+    std::mt19937 gen(rd());
+    std::uniform_int_distribution<int> dis(low, high);
+    return dis(gen);
+}
+
+void ROIAlignLayerTest::fillCoordTensor(std::vector<float>& coords, int height, int width,
```
Comment (iefode) [inline src/tests/functional/shared_test_classes/src/single_op/roi_align.cpp:56]: Hm. Coordinates? In this case why we use `float`? Looks like should be renamed

**Review by iefode — APPROVED** [review]

File: src/tests/functional/plugin/shared/include/single_op_tests/roi_align.hpp (hunk 1 above)
Comment (iefode) [reply src/tests/functional/plugin/shared/include/single_op_tests/roi_align.hpp:15]: Could you please check who is author and ask him? If the author would not provide correct answer, remove this case from lib. Thanks for your attention :)

Comment (vurusovs) [reply src/tests/functional/plugin/shared/include/single_op_tests/roi_align.hpp:15]: @iefode it's used in GPU tests, so I'll leave it as is

File: src/tests/functional/shared_test_classes/src/single_op/roi_align.cpp (hunk 2 above)
Comment (vurusovs) [reply src/tests/functional/shared_test_classes/src/single_op/roi_align.cpp:56]: I have no idea how it should be renamed, so will port as is. If you have some suggestions, I'm open to apply it

**Pull request merged**